meteor-app/stories/0-Welcome.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
This is just a storybook demo file - please delete

Reply:
it's deleted